lms/djangoapps/courseware/user_state_client.py

```diff
@@ -155,6 +175,29 @@ def set_many(self, username, block_keys_to_state, scope=Scope.user_state):
         else:
             user = User.objects.get(username=username)
 
+        evt_time = time()
+
+        def _ddog_increment(evt_name):
+            """
+            DataDog increment method.
+            """
+            dog_stats_api.increment(
+                'DjangoXBlockUserStateClient.set_many.{}'.format(evt_name),
+                timestamp=evt_time,
+                sample_rate=self.API_DATADOG_SAMPLE_RATE,
+            )
+
+        def _ddog_histogram(evt_name, value):
+            """
+            DataDog histogram method.
+            """
```

Comment:
This method is defined twice, would it be better make it a method on DjangoXBlockUserStateClient or in some kind of utils or helpers file that can be imported?